src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -272,6 +272,10 @@ private ExpressionSyntax Transform( ExpressionSyntax expr )
 			IdentifierNameSyntax identExpr => identExpr
 				.WithIdentifier( RemoveAsyncSuffix( identExpr.Identifier, optional: true ) ),
 
+			ImplicitObjectCreationExpressionSyntax implicitObjectCreationExpr => implicitObjectCreationExpr
+				.WithArgumentList( Transform( implicitObjectCreationExpr.ArgumentList ) )
+				.WithInitializer( MaybeTransform( implicitObjectCreationExpr.Initializer, Transform ) ),

```

Comment:
This is the same as `ObjectCreationSyntax` except that `ArgumentList` is not nullable for `ImplicitObjectCreationExpressionSyntax`, so we don't need the `MaybeTransform` wrapper.